osquery/config/plugins/http.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ *  Copyright (c) 2014, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <vector>
+#include <sstream>
+
+#include <osquery/config.h>
+#include <osquery/flags.h>
+#include <osquery/logger.h>
+#include <osquery/filesystem.h>
+#include <osquery/registry.h>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include "osquery/remote/requests.h"
+#include "osquery/remote/transports/http.h"
+#include "osquery/remote/serializers/json.h"
+
+#define MAX_TRIES 5
+
+namespace osquery {
+
+DECLARE_string(app_id);
+
+FLAG(string,
+     configuration_uri,
```

Comment:
Same, the existing `config_path` should be named `config_filesystem_path`, so maybe `config_enrollment_uri`?